packs/data/abomination-vaults-bestiary.db/drill-field-barbazu.json

```diff
@@ -1105,7 +1105,23 @@
                 "requirements": {
                     "value": ""
                 },
-                "rules": [],
+                "rules": [
+                    {
+                        "critical": false,
```

Comment:
Why no doubling on crit?

Reply:
I wasn’t sure that it should, since it’s an extra effect.